rs/tests/driver/src/driver/resource.rs

```diff
@@ -344,11 +345,11 @@ pub fn allocate_resources(
                         CreateVmRequest {
                             primary_image: ImageLocation::PersistentVolumeClaim {
                                 name: match req.primary_image.image_type {
-                                    ImageType::IcOsImage => {
-                                        format!("image-guestos-{}", tnet.image_sha)
-                                    }
                                     ImageType::PrometheusImage => "img-prometheus-vm".into(),
                                     ImageType::UniversalImage => "img-universal-vm".into(),
+                                    _ => {
+                                        format!("image-guestos-{}", tnet.image_sha)
```

Comment:
Maybe include "icos" in the name to indicate these are general IC-OS images:
```suggestion
                                        format!("image-icos-{}", tnet.image_sha)
```